ReactSkia/components/RSkComponentTextInput.h

```diff
@@ -40,13 +41,16 @@ class RSkComponentTextInput final : public RSkComponent {
   bool isTextInputInFocus_=false;
   bool secureTextEntry_=false;
   bool hasToSetFocus_=false;
+  bool showSoftInputOnFocus_=true;//To decide OnScreen KeyBoard to be used or not
```

Comment:
should this be atomic bool ?
since two threads will be accessing this variable 
main thread for updating and input/worker thread during key event

Reply:
This will be updated only by main thread on update property. Won't be accessed by worker thread

Reply:
AFAIK showSOftInputOnFocus_ is used in request/resignFromEditing Mode ... This is called in multiple place: onHandleKey -> input thread , onPaint -> mainThread , handleCommand -> mainThread and this variable is set in main thread

Reply:
Got it. Will Update.